hack/build-clusteradm-image.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/env bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Purpose: build a clusteradm image and push it to the registry.
+
+# Usage: $0 [--version|-v version] [--repo registry] [--platform platforms] [--verbose|-V] [-X]
+# Working directory does not matter.
+
+set -e
+
+clusteradm_version="" # ==> latest
+verbose="false"
+clusteradm_folder=clusteradm
+repo=quay.io/kubestellar
```

Comment:
In container image jargon, this is a "registry" rather than a "repo" (as correctly stated in the command line flags).